<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CustomView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>781</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Forwheel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Action">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player1 select a pawn</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerslot1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>600</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Player 1 Color: Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerslot2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>620</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>117</red>
          <green>80</green>
          <blue>123</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>117</red>
          <green>80</green>
          <blue>123</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Player 2 Color: Magenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="Errormessage">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="wheelbutton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>650</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Spin Wheel</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerslot3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>640</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>114</red>
          <green>159</green>
          <blue>207</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>114</red>
          <green>159</green>
          <blue>207</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="playerslot4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>660</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>226</green>
          <blue>52</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>138</red>
          <green>226</green>
          <blue>52</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Giveupturn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>650</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Give up movement</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>wheelbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spinWheel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>738</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Giveupturn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>giveUpMovement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>spinWheel()</slot>
  <slot>giveUpMovement()</slot>
 </slots>
</ui>
